package.json

```diff
@@ -30,62 +30,59 @@
     "focus-visible": "^5.2.0"
   },
   "devDependencies": {
-    "@alaskaairux/design-tokens": "^3.12.2",
-    "@alaskaairux/icons": "^4.22.1",
+    "@alaskaairux/design-tokens": "^3.13.0",
+    "@alaskaairux/icons": "^4.27.0",
     "@alaskaairux/webcorestylesheets": "^3.7.3",
     "@aurodesignsystem/eslint-config": "^1.3.0",
-    "@babel/eslint-parser": "^7.21.3",
-    "@commitlint/cli": "^17.4.4",
-    "@commitlint/config-conventional": "^17.4.4",
-    "@open-wc/testing": "^3.1.7",
-    "@rollup/plugin-node-resolve": "^15.0.1",
-    "@semantic-release/changelog": "^6.0.2",
+    "@babel/eslint-parser": "^7.21.8",
+    "@commitlint/cli": "^17.6.3",
+    "@commitlint/config-conventional": "^17.6.3",
+    "@open-wc/testing": "^3.1.8",
+    "@rollup/plugin-node-resolve": "^15.0.2",
+    "@semantic-release/changelog": "^6.0.3",
     "@semantic-release/git": "^10.0.1",
-    "@semantic-release/npm": "^9.0.2",
-    "@web/dev-server": "^0.1.36",
-    "@web/test-runner": "^0.15.1",
-    "@webcomponents/webcomponentsjs": "^2.7.0",
+    "@semantic-release/npm": "^10.0.3",
+    "@web/dev-server": "^0.2.1",
+    "@web/test-runner": "^0.16.1",
+    "@webcomponents/webcomponentsjs": "^2.8.0",
     "autoprefixer": "^10.4.14",
     "compression": "^1.7.4",
     "concat": "^1.0.3",
-    "concurrently": "^7.6.0",
+    "concurrently": "^8.0.1",
     "copyfiles": "^2.4.1",
-    "core-js": "^3.29.1",
-    "eslint": "^8.36.0",
-    "eslint-plugin-jsdoc": "^40.1.0",
+    "core-js": "^3.30.2",
+    "eslint": "^8.40.0",
+    "eslint-plugin-jsdoc": "^44.0.0",
     "focus-visible": "^5.2.0",
     "fs-extra": "^11.1.1",
     "husky": "^8.0.3",
     "lodash": "^4.17.21",
     "markdown-magic": "^2.6.1",
-    "nodemon": "^2.0.21",
+    "nodemon": "^2.0.22",
     "npm-run-all": "^4.1.5",
-    "postcss": "^8.4.20",
-    "postcss-custom-properties": "^13.1.4",
-    "postcss-discard-comments": "^5.1.2",
-    "postcss-rem-to-pixel": "^4.1.2",
-    "postcss-remove-rules": "^1.0.0",
+    "postcss": "^8.4.23",
+    "postcss-discard-comments": "^6.0.0",
     "postcss-selector-replace": "^1.0.2",
     "prismjs": "^1.29.0",
-    "rollup": "^3.20.0",
+    "rollup": "^3.21.5",
     "rollup-plugin-serve": "^2.0.2",
-    "sass": "^1.59.3",
-    "semantic-release": "^20.1.3",
-    "stylelint": "^15.3.0",
+    "sass": "^1.62.1",
```

Comment:
Please add the following for properly using Sass

https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/package.json#L75-L79